### ISSUE TITLE
[Bug] "Using $this when not in object context" when i use related model

### ISSUE THREAD

**Issue by pana1990** [codemix/yii2-excelexport#21]
I get the following error "Using $this when not in object context" when i use related model.

Example code : 
```php
$file = \Yii::createObject([
                'class' => ExcelFile::class,
                'sheets' => [
                    'MoneyMovements' => [
                        'class' => ActiveExcelSheet::class,
                        'query' => User::find(),
                        'attributes' => [
                            'id',
                            'address.street', // related model
                        ],
                    ],
                ],
            ]);
```

![screenshot_1](https://user-images.githubusercontent.com/6630197/38135109-96950784-3416-11e8-9e4e-e1affc0b0d85.png)

**Comment by pana1990** [comment]
@mikehaertl check my pr please https://github.com/codemix/yii2-excelexport/pull/22

**Comment by mikehaertl** [comment]
@pana1990 Just released 2.7.1 containing this fix. Thanks for finding this bug.
